homeassistant/components/sensor/rfxtrx.py

```diff
@@ -29,25 +31,46 @@ def setup_platform(hass, config, add_devices_callback, discovery_info=None):
     """Setup the RFXtrx platform."""
     from RFXtrx import SensorEvent
 
+    sensors = []
+    devices = config.get('devices', None)
```

Comment:
Instead of having to do `if devices` you should provide an empty `dict` as default parameter instead of `None`.

```python
for entity_id, entity_info in config.get('devices', {}).items():
```